provider/backup_utils.py

```diff
@@ -405,6 +405,21 @@ def create_image_by_params(vm, params, image_name):
     return image
 
 
+def create_image_with_data_file(vm, params, image_name):
+    """Create image with data_file"""
```

Comment:
Add comments here to describe what's the returned, here we return a list containing two StorageVolume objects

Reply:
Have upated, thanks zhenchao.